mmv1/templates/terraform/state_migrations/filestore_instance.go.erb

```diff
@@ -138,8 +138,8 @@ addresses reserved for this instance.`,
         Type:         schema.TypeString,
         Required:     true,
         ForceNew:     true,
-        ValidateFunc: validation.StringInSlice([]string{"TIER_UNSPECIFIED", "STANDARD", "PREMIUM", "BASIC_HDD", "BASIC_SSD", "HIGH_SCALE_SSD", "ZONAL"}, false),
```

Comment:
This is a cached version of the state at the time the state migration was written - it shouldn't be updated.

Reply:
Ack, state migration modification reverted.